app/controllers/leafs_controller.rb

```diff
@@ -14,10 +14,10 @@ def create
 
     respond_to do |format|
       if @leaf.save
-        flash[:success] = "Leaf created!"
+        flash[:success] = 'Leaf created!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.